static/css/homepage.css

```diff
@@ -0,0 +1,208 @@
+.about_button
+{
+	margin-left: 50%;
+}
+.abouttext
+{
+	padding-bottom: 5%;
+}
+.layoutbutton_image
+{
+	margin-top: 15%;
+}
+.marginb_w
+{
+	margin-bottom: 3%;
+}
+.marginl_w
+{
+	margin-left: 10%;
+}
+.marginr_w
+{
+	margin-right: 10%;
+}
+.yellowbackground
+{
+	background-color: #F8F5A7;
+}
+.logotext
+{
+	margin-left: 10%;
+	margin-top: 10%;
+}
+.arrowtl
+{
+	color: #172B59;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-size: 100px;
+	font-weight: 200;
+	margin-bottom: 5%;
+	margin-right: 10%;
+}
+.arrowtr
+{
+	color: #172B59;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-size: 100px;
+	font-weight: 200;
+	margin-left: 15%;
+	padding-bottom: 5%;
+}
+.arrowl
+{
+	color: #172B59;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-size: 100px;
+	font-weight: 200;
+	margin-bottom: 5%;
+	margin-right: 5%;
+}
+.arrowr
+{
+	color: #172B59;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-size: 100px;
+	font-weight: 200;
+	margin-bottom: 5%;
+	margin-left: 5%;
+}
+.icon48
+{
+	color: #172B59;
+	font-size: 68px;
+}
+.icon38
+{
+	color: #172B59;
+	font-size: 38px;
+}
+.icon38w
+{
+	background-color: #172B59;
+	color: #DEE8F2;
+	font-size: 38px;
+	margin-left: 1%;
+	margin-right: 1%;
+}
+.hero-area
+{
+	background: url('/static/images/homepage/hero_bg.jpg')no-repeat;
```

Comment:
"/static/images/homepage/hero_bg.jpg" Use of external graphics and images can be tricky. We don't know the copyrights of these. We can take it out until we design something on our own.